tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -57,42 +61,66 @@ setup() {
     k8s_create_pod "$kata_pod_with_nydus_config"
 }
 
-@test "Test we can pull a large image inside the guest" {
-    [[ " ${SUPPORTED_NON_TEE_HYPERVISORS} " =~ " ${KATA_HYPERVISOR} " ]] && skip "Test not supported for ${KATA_HYPERVISOR}."
-    skip "This test requires large memory, which the encrypted memory is typically small and valuable in TEE. \
-          The test will be skiped until https://github.com/kata-containers/kata-containers/issues/8142 is addressed."
-    kata_pod_with_nydus_config="$(new_pod_config "$large_image" "kata-${KATA_HYPERVISOR}")"
-    set_node "$kata_pod_with_nydus_config" "$node"
-    set_container_command "$kata_pod_with_nydus_config" "0" "sleep" "30"
+@test "Test we cannot pull an image that exceeds the memory limit inside the guest" {
+    # The image pulled in the guest will be downloaded and unpacked in the `/run/kata-containers/image` directory. 
+    # However, by default, systemd allocates 50% of the available physical RAM to the `/run` directory using a `tmpfs` filesystem. 
+    # It means that if we run a kata container with the default configuration (where the default memory assigned for a VM is 2048 MiB), 
+    # `/run` would be allocated around 1024 MiB. Consequently, we can only pull images up to 1024 MiB in the guest. 
+    # However, the unpacked size of image "rust:latest" is 1.41GB. 
```

Comment:
why don't keep using quay.io/confidential-containers/test-images:largeimage which is larger than 1.41GB, so will fail as well?

Reply:
Because the default pull timeout is 60s and it would be timeout if we pull `quay.io/confidential-containers/test-images:largeimage` in the case `Test we can pull an image inside the guest using trusted storage`.

Reply:
ah, ok. so it should large enough to get pulled before the deadline. Thanks for the explanation!